docs/resources/Guild.md

```diff
@@ -643,7 +643,7 @@ Returns a list of guild [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object
 
 ## Create Guild Channel % POST /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/channels
 
-Create a new [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object for the guild. Requires the `MANAGE_CHANNELS` permission. Returns the new [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object on success. Fires a [Channel Create](#DOCS_TOPICS_GATEWAY/channel-create) Gateway event.
+Create a new [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object for the guild. Requires the `MANAGE_CHANNELS` permission. If setting permission overwrites, only permissions your bot has in the guild can be allowed/denied. Setting `MANAGE_ROLES` permission in channels is only possible for guild administrators. Returns the new [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object on success. Fires a [Channel Create](#DOCS_TOPICS_GATEWAY/channel-create) Gateway event.
```

Comment:
> Setting `MANAGE_ROLES` permission in channels is only possible for guild administrators

This doesn't seem to be the case in my testing. The bot has MANAGE_ROLES in the category and then can also create channels with that permission.

```
Bot Permissions (in category): [MANAGE_CHANNEL, MESSAGE_ADD_REACTION, VIEW_CHANNEL, MESSAGE_READ, MESSAGE_WRITE, MESSAGE_TTS, MESSAGE_EXT_EMOJI, VOICE_STREAM, MANAGE_ROLES]
Override(420321485757087746): allow=[MANAGE_CHANNEL, MANAGE_ROLES] deny=[]
Override(658513573412077589): allow=[VOICE_USE_VAD] deny=[MANAGE_ROLES]
```

Reply:
There's some nuances not documented to power channel cloning in guilds (as a tempfix), and I suspect that may be the reason you see this behavior. Are there other channels in your test guild which share identical permissions that you have permission in?

Reply:
I have a lot of channels, so that is very possible indeed. Would it make sense to also allow users/bots to sync permissions to the category without having administrator permissions? It seems illogical to require administrator just to be able to create sync'd channels. Maybe that would make the rules too complex though, unless a simpler way to created sync channels is implemented like `lock_permissions` in #1776.

I'd love to see a more nuanced permission in the future since administrator seems a bit too powerful of a permission for this kinda problem.